plugins/plugin-chart-echarts/src/Timeseries/EchartsTimeseries.tsx

```diff
@@ -32,10 +35,55 @@ export default function EchartsTimeseries({
   labelMap,
   selectedValues,
   setDataMask,
+  legendData = [],
 }: TimeseriesChartTransformedProps) {
+  const { emitFilter, stack } = formData;
+  const echartRef = useRef<EchartsHandler | null>(null);
+  const lastTimeRef = useRef(Date.now());
+  const lastSelectedLegend = useRef('');
+  const clickTimer = useRef<ReturnType<typeof setTimeout>>();
+
+  const handleDoubleClickChange = useCallback((name?: string) => {
+    const echartInstance = echartRef.current?.getEchartInstance();
+    if (!name) {
+      echartInstance?.dispatchAction({
+        type: 'legendAllSelect',
+      });
+    } else {
+      legendData.forEach(datum => {
+        if (datum === name) {
+          echartInstance?.dispatchAction({
+            type: 'legendSelect',
+            name: datum,
+          });
+        } else {
+          echartInstance?.dispatchAction({
+            type: 'legendUnSelect',
+            name: datum,
+          });
+        }
+      });
+    }
+  }, []);
+
+  const getModelInfo = (target: ViewRootGroup, globalModel: GlobalModel) => {
```

Comment:
This function involves the low-level processing logic of echarts and serves to obtain the series model.